content/en/docs/concepts/configuration/secret.md

```diff
@@ -684,7 +684,7 @@ for information on referencing service account credentials from within Pods.
 
 ### Docker config Secrets
 
-You can use one of the following `type` values to create a Secret to
+You **MUST** use one of the following `type` values to create a Secret to
```

Comment:
```suggestion
You **must** use one of the following `type` values to create a Secret to
```

bold is enough for emphasis.

Reply:
Alternatively,
```suggestion
If you are creating a Secret to store credentials for accessing a container image registry,
you must use one of the following `type` values for that Secret.
```

Reply:
In my opinion, I prefer [this](https://github.com/kubernetes/website/pull/41760/files#r1242021712) ⬆️ suggestion or the original